### ISSUE TITLE
Default landscape settings are wrong

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#48]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/48) by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

A few years ago someone went and upped all the values in the default client config. The default LandscapeTileNear is now way too high, as this value controls the border between the nar and far tiles for the landscape. The near tiles need to be rendered without fog, which causes an excessive value for LandscapeTileNear to give a very cheap looking landscape in the distance. LandscapeThreshold does not visibly improve after 2000, so this should be reverted as well.

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Original values


```
#!c

// *** LANDSCAPE
LandscapeTileNear		= 50.000000;
LandscapeTileNear_min	= 20.000000;
LandscapeTileNear_max	= 100.000000;
LandscapeTileNear_step	= 10.0;
LandscapeTileNear_ps0	= 20.0;
LandscapeTileNear_ps1	= 40.0;
LandscapeTileNear_ps2	= 50.0;
LandscapeTileNear_ps3	= 80.0;

// NB: threshold is inverted ULandscape::setThreshold(), to be more intelligible
LandscapeThreshold		= 1000.0;
LandscapeThreshold_min	= 100.0;	// Low quality => 0.01 threshold
LandscapeThreshold_max	= 2000.0;	// High quality => 0.0005 threshold
LandscapeThreshold_step	= 100.0;
LandscapeThreshold_ps0	= 100.0;
LandscapeThreshold_ps1	= 500.0;
LandscapeThreshold_ps2	= 1000.0;
LandscapeThreshold_ps3	= 2000.0;
```

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Notice the fog mismatch with the purple tent and the trees when this setting is set beyond it's intended value.

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Fix issue #48, landscape quality issues caused by incorrect defaults
